rdp controller described, .gitbook/assets folder renamed to -assets (#18)
G1ANT-Robot/G1ANT.Manual: appendices/rdp-controller.md

# RDP Controller

RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.

> Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.

Reviewer: Try something like that:
"Our RDP Controller is a tool allows running automations with G1ANT.Robot on a remote PC without disturbance even when you close you RDP client and disconnect RDP session..."

Author: The sense is almost the same, only my description is more detailed and probably more comprehensive for people who are not that advanced in IT.

Author: ```suggestion
Our RDP Controller is a tool for running G1ANT.Robot automations on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection client. The script will continue the execution without disturbance.
```

In order to start using the RDP Controller, you have to follow these steps:

> 1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.

Reviewer 2: `[this file]()` 
there's no path to this file

Author: don't you say :smile:
look at my first comment

Author: ```suggestion
1. Download and run [this file](../-assets/RDP local machine.reg) on your local Windows system, answering *Yes* to the security warning message.
```

2. Launch Remote Desktop Connection tool in Windows: type `remote` in the Windows Search box (*Windows 7:* click Start Menu, then use the search box; *Windows 8 and above:* click the Search icon on the taskbar) and click the name of the app in the search results. You can also select the tool in the Accessories (Windows 7) or Windows Accessories (Windows 8 and above) folder of the Start menu (click All Programs first in case of Windows 7).

>    ![](G1ANT.Manual/-assets/rdc-start.jpg)

Reviewer 2: incorrect path, should be: `![](-assets/rdc-start.jpg)`

Author: ```suggestion
   ![](../-assets/rdc-start.jpg)
```

3. Enter username and password of the user on the remote machine.

> 4. Launch G1ANT.Robot, go to `Tools/Installers` menu and select `RDP Controller`.

Reviewer: Add information that to install RDP Controller G1ANT.Robot needs admin privileges.

Author: ```suggestion
4. Launch G1ANT.Robot, go to `Tools/Installers` menu and select `RDP Controller`. You will need administrator rights to install the RDP Controller.
```

5. Repeat step 3. and enter these credentials in the RDP Configuration dialog box.

6. Click Yes in the security warning message box. Wait until the command line installer finishes its work and when it does, press any key to close the command line window.

   To check if the RDP Controller was successfully installed, open Windows Task Manager (press **Ctrl+Shift+Esc**) go to Services tab and look for *Disconnected Rdp Session Replacer* service. If it’s there and running, the RDP Controller is ready for use.

>    ![](G1ANT.Manual/-assets/rdp-task.jpg)

Reviewer 2: incorrect path

Reviewer 2: should be:  `![](-assets/rdp-task.jpg)`
it's always good to check if something is visible on github by clicking view file

Author: ```suggestion
   ![](../-assets/rdp-task.jpg)
```

7. Run your script.

Now you can minimize the remote desktop window or close it altogether.

> **Note:** the two Registry keys, which are added by our file installed in step 1, allow minimizing the remote desktop window without affecting the robot and its script execution.

Reviewer: Please add note that without those registry entries automation will not work correctly when RDP client will be minimised

Author: ```suggestion
**Note:** the two Registry keys, which are added by our file installed in step 1, allow minimizing the remote desktop window without affecting the robot and its script execution. Without these registry entries the automation will not work correctly when the RDP client is minimized.
```
